ace.d.ts

```diff
@@ -227,6 +227,7 @@ export namespace Ace {
     value: string;
     session: EditSession;
     relativeLineNumbers: boolean;
+    enableMultiselect: boolean;
```

Comment:
I might be missing something, but what is this option being used for?

Reply:
This is an existing option, which is missing from TS declaration. This option enables "ctrl+click" to enable multiple cursors.